test/integ/matMul/src/matMul.cpp

```diff
@@ -228,10 +228,16 @@ TEMPLATE_LIST_TEST_CASE("matMul", "[matMul]", TestAccs)
     auto bufCAcc = alpaka::allocBuf<Val, Idx>(devAcc, extentC);
 
     // Copy Host -> Acc.
-    alpaka::memcpy(queueAcc, bufAAcc, bufAHost);
-    alpaka::memcpy(queueAcc, bufBAcc, bufBHost);
-    alpaka::wait(queueHost);
-    alpaka::memcpy(queueAcc, bufCAcc, bufCHost);
+    std::cout << "Input copy time:  "
+              << alpaka::test::integ::measureRunTimeMs(
+                     [&]
+                     {
+                         alpaka::memcpy(queueAcc, bufAAcc, bufAHost);
+                         alpaka::memcpy(queueAcc, bufBAcc, bufBHost);
+                         alpaka::memcpy(queueAcc, bufCAcc, bufCHost);
+                         alpaka::wait(queueHost);
```

Comment:
I moved the wait to the last command to have the total copy time of all 3 buffers.

Reply:
Ok, that was a bug :D The `wait` ensures that the `memset` on `bufCHost` finished before copying from it.